app/src/main/java/org/simple/clinic/summary/teleconsultation/contactdoctor/DoctorListItem.kt

```diff
@@ -0,0 +1,69 @@
+package org.simple.clinic.summary.teleconsultation.contactdoctor
+
+import androidx.recyclerview.widget.DiffUtil
+import io.reactivex.subjects.Subject
+import kotlinx.android.synthetic.main.list_contact_doctor.*
+import org.simple.clinic.R
+import org.simple.clinic.summary.teleconsultation.contactdoctor.DoctorListItem.Event.ContactMode.SMS
+import org.simple.clinic.summary.teleconsultation.contactdoctor.DoctorListItem.Event.ContactMode.WhatsApp
+import org.simple.clinic.summary.teleconsultation.sync.MedicalOfficer
+import org.simple.clinic.widgets.ItemAdapter
+import org.simple.clinic.widgets.recyclerview.ViewHolderX
+
+data class DoctorListItem(
+    private val name: String,
+    private val number: String
+) : ItemAdapter.Item<DoctorListItem.Event> {
+
+  companion object {
+    fun from(medicalOfficers: List<MedicalOfficer>): List<DoctorListItem> {
+      return medicalOfficers
+          .map {
+            DoctorListItem(
+                name = it.fullName,
+                number = it.phoneNumber
+            )
+          }
+    }
+  }
+
+  class DiffCallback : DiffUtil.ItemCallback<DoctorListItem>() {
+    override fun areItemsTheSame(oldItem: DoctorListItem, newItem: DoctorListItem): Boolean {
+      return oldItem.number == newItem.number
```

Comment:
I feel we should add the Medical officer UUID to the model and use that to check whether the items are the same since there could be scenarios where the doctor updates their teleconsultation number on the dashboard.

Reply:
Yeah, good point. I will add `doctorId`